# SYMMF_net
Microbiome network analysis with NMF


Title

Identification of functional microbial modules through network-based analysis of meta-microbial features using matrix factorization.


Abstract

 As the microbiome is composed of a variety of commensal, symbiotic, and pathogenic microbial interactions, it is imperative in microbiome research to identify a microbial sub-community that collectively conducts a specific function. However, current methodologies have been highly limited to analyzing conditional abundance changes of individual microorganisms, without considering group-wise collective microbial features. To overcome this limitation, we developed a network-based method using non-negative matrix factorization (NMF) to identify functional meta-microbial features (MMFs) that, as a group, better discriminate specific environmental conditions of samples using 16s rRNA sequencing data. 
 As proof of concept, large scale human microbiome data collected from different body parts were used to identify body-part specific MMFs by applying NMF. The results showed that a small number of MMFs was adequately valid to represent the sample information with higher clustering purity than original raw data. Moreover, the permutation test for MMFs enabled us to identify highly discriminative MMFs on sample classes, called synergistic MMFs (SYMMFs). Subsequent analysis of SYMMFs suggested that multiple microorganisms involved in SYMMFs contain useful sample-specific microbial features as functional combinations. Finally, we constructed a SYMMF-based microbial interaction network (SYMMF-net) by integrating all of the SYMMF information. Network analysis revealed core microbial modules closely related to critical sample properties. Consistent results were also found when the method was applied to various disease-associated microbiome data.
 The developed method interprets high-dimensional microbiome data by identifying functional microbial modules on sample properties and intuitively representing their systematic relationships via a microbial network. We anticipate that this method is generally applicable in analyzing various microbiome data and facilitates understanding of unknown disease-related microbial relationships.


